### DISCUSSION TITLE
Graph parser/exporter

### DISCUSSION THREAD

**Discussion by romanzdk** [apache/airflow#34806] [Q&A]
I would like to export Airflow DAG as graph image similarly as CLI command `show dag` works but _offline_ - meaning I do not want to exec into running pod to run the command. I was thinking of some parser that would parse the python module file and based on that export some svg. Is there any tool for that?

**Comment by Taragolis** [comment]
Looks like you need to install airflow locally into the venv with the same dependencies and run required cli command locally.
